### PR TITLE
Add upstream changes

### PR DESCRIPTION
**Pull request by mcdonnnj**
Merge in upstream changes in skeleton-ansible-role.

### CODE REVIEW

**Comment by jsf9k** [comment]
If you have build problems after pulling in the buster Docker image, it's because that image hasn't been recreated since buster was officially released.  The `apt-get` repos have changed from "testing" to "stable", and so `apt-get` complains about this.

See debuerreotype/docker-debian-artifacts#76, which is where the official images on Docker Hub come from.

If the images don't get updated soon we can [add some Ansible code to temporarily work around the issue](https://superuser.com/questions/1456989/how-to-configure-apt-in-debian-buster-after-release), but I'd rather not have to do that.

File: meta/main.yml [hunk 1]
```diff
@@ -9,6 +9,10 @@ galaxy_info:
     - name: Debian
       versions:
         - stretch
+        - buster
+    - name: Amazon
```
Comment (jsf9k) [inline meta/main.yml]: I don't think we want the Amazon foo here.

File: molecule/default/molecule.yml [hunk 2]
```diff
@@ -8,6 +8,10 @@ lint:
 platforms:
   - name: debian9
     image: debian:stretch-slim
+  - name: debian10
+    image: debian:buster-slim
+  - name: amazon2018.03
```
Comment (jsf9k) [inline molecule/default/molecule.yml]: See previous comment.

**Review by jsf9k — CHANGES_REQUESTED** [review]

**Comment by jsf9k** [comment]
Also looks like this PR may have lost contact with its TravisCI builds.  You can restart the branch and PR builts or wait until they resolve themselves overnight.

**Review by felddy — APPROVED** [review]
Praise the lord and pass the ammunition!

**Review by jsf9k — APPROVED** [review]
:+1:

**Comment by jsf9k** [comment]
Just got an email that TravisCI is having issues. So you may have to wait until that is resolved for your PRs to clear.

**Pull request merged**